dataprocspawner/spawner.py

```diff
@@ -907,30 +917,39 @@ def _build_cluster_config(self, cluster_data=None):
               init_actions + cluster_data['config']['initialization_actions']
       )
 
+      if 'labels' in cluster_data:
+        for key in list(cluster_data['labels']):
+          cluster_data['labels'][key] = cluster_data['labels'][key]
```

Comment:
This operation seems like a no-op. What's the intent here?

Reply:
Well, my thought was, that's how you, guys, ensure the cluster definition is formed properly - redefining config key-value pairs. I can remove that. What would you suggest in this case?

Reply:
I think Jerry means that ```cluster_data['labels'][key] = cluster_data['labels'][key]``` seems to do something like ```a = a```. 
Generally, when we set a value, we use another value. Maybe you can point us to a line where we do something similar?

Reply:
OK, maybe that's my loss, I will remove this code and test it. I'll resolve this conversation as soon as I make updates

Reply:
The latest code changes can't be tested due to an issue described here https://github.com/GoogleCloudDataproc/jupyterhub-dataprocspawner/issues/22
Please, pay attention.

Reply:
This should be fixed now. Can you try again?